src/Polymesh.ts

```diff
@@ -255,6 +256,17 @@ export class Polymesh {
     return reserveTicker.prepare(args, this.context);
   }
 
+  /**
+   * Check if a ticker is available
+   */
+  public async isTickerAvailable(args: { ticker: string }): Promise<boolean> {
+    const { ticker } = args;
```

Comment:
No need to destructure here, you could just pass `args` to the `TickerReservation` constructor